### ISSUE TITLE
[TwigComponent] Twig ExtensionSet is initialized to soon since ux-twig-component 2.8.0

### ISSUE THREAD

**Issue by nexxome** [symfony/ux#835]
### Symfony version(s) affected

6.2.10

### Description

The problem occurred in combination with https://github.com/php-translation/symfony-bundle and the translation:extract command.
I'm getting this exception:

>  Unable to register extension "Translation\Extractor\FileExtractor\TwigFileExtractor" as extensions have already been initialized.

In `Symfony\UX\TwigComponent\DependencyInjection\TwigComponentExtension` the service `twig.configurator.environment` gets decorated by `Symfony\UX\TwigComponent\Twig\TwigEnvironmentConfigurator` and there the `Symfony\UX\TwigComponent\Twig\ComponentLexer` is initialized which will lead to the initialization of `Twig\ExtensionSet`:

Symfony\UX\TwigComponent\Twig\TwigEnvironmentConfigurator (31):
`$environment->setLexer(new ComponentLexer($environment));`

Twig\Lexer (457):
`array_keys($this->env->getUnaryOperators()),`

Twig\ExtensionSet (396):
```
    public function getUnaryOperators(): array
    {
        if (!$this->initialized) {
            $this->initExtensions();
        }

        return $this->unaryOperators;
    }
```

### How to reproduce

1. Add symfony/ux-twig-component and php-translation/symfony-bundle as dependency
2. Run `php bin/console translation:extract`

**Comment by weaverryan** [comment]
Hmm, yes, I had hit this too in a test - though we were doing some crazy things in the test, so I just worked around it. Pr open at #836 - though it's ugly. This feels like a Twig edge bug that we were the first to experience.

**Comment by weaverryan** [comment]
This is now fixed in Twig! 2.15.5 contains it. It looks like 3.6.0 WILL fix it also, but it's not released yet. You can bring in the 3.x-dev branch in your project until then to work around the issue.

Cheers!

**Comment by nexxome** [comment]
Thank you very much for the fast fix! ❤️
